.github/integration/scripts/make_sda_credentials.sh

```diff
@@ -121,6 +121,22 @@ if [ ! -f "/shared/sync.sec.pem" ]; then
     /shared/crypt4gh generate -n /shared/sync -p syncPass
 fi
 
+if [ ! -f "/shared/rotatekey.sec.pem" ]; then
+    echo "creating rotatekey crypth4gh key"
+    /shared/crypt4gh generate -n /shared/rotatekey -p rotatekeyPass
+fi
+
+# register the rotation key in the db
+resp=$(psql -U postgres -h postgres -d sda -At -c "SELECT description FROM sda.encryption_keys;")
+if ! echo "$resp" | grep -q 'this is the new key to rotate to'; then
+    rotateKeyHash=$(cat /shared/rotatekey.pub.pem | awk 'NR==2' | base64 -d | xxd -p -c256)
```

Comment:
```suggestion
    rotateKeyHash=$(awk 'NR>1 && !/-----/' /shared/rotatekey.pub.pem | base64 -d | xxd -p -c256 | tr -d '\n')
```

More robust parsing of the key
1. In case base64 content spans for multiple lines 
2. In case the output of `xxd -p -c256` may truncated or reformatted unexpectedly.

Reply:
I over considered the generality. You are right, no need to change if we stick to c4gh.